### PR TITLE
Clarify APIs and admin role reqs in sub policies in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/22781

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good! I fixed a couple of older typos that I saw. Also, for consistency, we try to put the colon inside of the asterisks in *Note:*. Not something to take time changing, but I went ahead and changed them in here. I think you were following what was there, but we want use the *Note:* when you happen to notice it. Thanks!

File: governance/deploy_gitops.adoc [hunk 1]
```diff
@@ -141,6 +141,20 @@ View the following explanations for each argument:
...
 
+*Note:* To deploy policies with subscriptions, complete the following steps:
+
+. Bind the `open-cluster-management:subscription-admin` ClusterRole to the user creating the subscription.
+. If you are using an allow list in the subscription, include the following entries:
+
++
+[source,yaml]
+----
+  allow:
```
Comment (gparvin) [inline governance/deploy_gitops.adoc]: I think we would want to show an allow list like this:

```
    - apiVersion: policy.open-cluster-management.io/v1
      kinds:
        - "*"
    - apiVersion: policy.open-cluster-management.io/v1beta1
      kinds:
        - "*"
    - apiVersion: apps.open-cluster-management.io/v1
      kinds:
        - PlacementRule
    - apiVersion: cluster.open-cluster-management.io/v1beta1
      kinds:
        - Placement

```

**Review by gparvin — COMMENTED** [review]
This looks good -- I just had the one comment about the allow list update.  Thanks

**Comment by dockerymick** [comment]
Remember to also make this update for 2.6 as well

**Pull request merged**